### PR TITLE
fixes #23 - updated js libs to latest version of 1.x compat

### LINKED ISSUE #23: keep table in place
The table from units/?view=table shouldn't snap under the header when scrolling down and causing the header to float.

### PR DESCRIPTION
**Pull request by ktiedt**
I have confirmed fancybox works on the pages its used in, it was the only lib that threw any errors from what I've seen but you are more inclined to know odd pages than I am ;)

### CODE REVIEW

File: blog/2017-03-15/index.php [hunk 1]
```diff
@@ -26,8 +26,8 @@
 require_once("../../header.php");
 ?>
 
-<link rel="stylesheet" href="<?php echo $site_root; ?>media/fancybox/jquery.fancybox-1.3.4.css" media="screen">
-<script src="<?php echo $site_root; ?>jquery.min.js"></script>
```
Comment (DreadKnight) [inline blog/2017-03-15/index.php]: It's better to have this lib locally for offline development. CloudFlare does it's magic and uses the CDN for it anyway.

Comment (ktiedt) [inline blog/2017-03-15/index.php]: I would love to see documentation on this magic CDN switch, I see nothing about it anywhere when searching.

Comment (DreadKnight) [inline blog/2017-03-15/index.php]: @ktiedt Can't pinpoint the exact place mentioning that atm, as they kept switching website, but if you look at the current website source code and search for jquery, you'll see that it's served from a CDN ajax.googleapis.com/ajax/libs/jquery/1.7.2/jquery.min.js, even though it should come from elsewhere (OpenShift server), though, it's a Google CDN, which might cause cause issues in China, as we were talking about on Discord server, sigh xD

Comment (DreadKnight) [inline blog/2017-03-15/index.php]: It's kinda silly that CF doesn't uses its own damn CDN, like you did. Strange.

**Pull request merged**